### PR TITLE
Name change to Clockwork achievements

### PR DESCRIPTION
**Pull request by Xoxeyos**
Achievement name changed so there are no duplicate clockwork names.

Eminence Achievement : Ratvar's Hand ( Sorta Lame, I know. Tell me if you come up with a better one.)

Cultist Achievement : Clocking In

🆑
rscadd: Name changes to CW Cultists.
/🆑

### CODE REVIEW

**Comment by overlord12345** [comment]
Maybe right hand man or ratvars right hand man

File: code/datums/achievements/achievements.dm [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 /datum/achievement/bubblegum
 	name = "Kick Ass and Chew Bubblegum"
-	desc = "Kill Bubblegum, the king of slaughter demons." //Should be improved(?)
```
Comment (Amelia0010) [inline code/datums/achievements/achievements.dm:16]: mane

Comment (Bibby0110) [reply code/datums/achievements/achievements.dm:16]: mane

Comment (Xoxeyos) [reply code/datums/achievements/achievements.dm:16]: resolved

**Pull request merged**